code/index.html

```diff
@@ -3,28 +3,91 @@
+        <img class="hero-image" src="https://images.unsplash.com/photo-1605812830455-2fadc55bc4ba?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" alt="hero image" />     
+      </picture>
+     <h1 class="hero-title">Welcome to Dad's Corner</h1>
+     <div class="hero-menu">
+       <nav class="topnav">
+         <a href="#home">Home</a>
+         <a href="#news">News</a>
+         <a href="#contact">Contact</a>
+         <a href="#about">About</a>
+       </nav>
+      </div>
+   </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<!-- Intro paragraph -->
+ 
+   <section class="introduction">
+      <h2>Are you a dad on paternity leave? This is the forum for you! Meet other dads, share experiences, have fun play-dates - it all happens here!</h2>
+   </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+<!-- The accordion section -->
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+   <section class="information">
+     <div class="accordion" class="container">
+       <h3>FAQ</h3>
+       <div class="question" id="section1"><h2>What is Dad's Corner?</h2></div>
+       <div class="answer" id="answer1"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+       <div class="question" id="section2"><h2>How do I create an account?</h2></div>
+       <div class="answer" id="answer2"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+       <div class="question" id="section3"><h2>How much does it cost?</h2></div>
+       <div class="answer" id="answer3"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,</p></div>
+       <div class="question" id="section4"><h2>What are the benefits</h2></div>
+       <div class="answer" id="answer4"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut  labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+     </div>
+
+ <!-- The right image -->
+
+     <div class="dad-son-img" class="container"></div>
+
+ <!-- The form -->
+
+     <form class="container" action="http://httpbin.org/anything" method="POST">
+        <h3>Sign in here</h3>
+        <div class="personal-information">
+           <label for="fname">First name</label>
+          <input type="text" id="fname" name="firt-name" value="firstname" minlength="3" maxlength="10" placeholder="Your name.." required>
```

Comment:
leave the value empty, the place holder is what should guide the user to put in the right content in the input. With value you 're population the form with "firstname" "lastname" and so on.